### PR TITLE
create a header with style

### PR DESCRIPTION
**Pull request by suhail-radwan**
Relates #3

### CODE REVIEW

**Review by reemsh2000 — APPROVED** [review]
good job

**Review by emanjaber — APPROVED** [review]
good job Suhail
keep on ^^

File: index.html [hunk 1]
```diff
@@ -12,6 +12,25 @@
...
+            <ul class="header_nav_list">
+              <li class="list_item_active"><a href="./index.html">Home</a></li>
+              <li><a href="./about.html">About Us</a></li>
+              <li><a href="#">Gallery</a></li> 
+            </ul>
+          </nav>
+        </div>
+      </header>
+    
+      <div class="banner_area">
```
Comment (sara219) [inline index.html:28]: use semantic HTML element

File: css/media.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  .banner_area p {
+    font-size: 9vw;
+  }
+}
+
+@media screen and (max-width: 900px) {
+    .header_nav {
+      font-size: 20px;
+      justify-content: space-between;
+    }
```
Comment (sara219) [inline css/media.css:35]: you forget the closing `} `for the media query

File: css/media.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+@media screen and (max-width: 768px) {
+  .header_container {
+    flex-direction: column;
+    text-align: center;
```
Comment (sara219) [inline css/media.css:4]: your media query doesn't work, it looks like this 

Screenshot from 2021-03-03 11-21-01

**Review by sara219 — CHANGES_REQUESTED** [review]

**Pull request merged**